packages/design-system/src/styles/components/_Button.scss

```diff
@@ -26,8 +26,9 @@
   &.ds-c-button--hover,
   &:hover &:focus,
   &.ds-c-button--hover &.ds-c-button--focus {
-    border-color: $color-primary-darker;
-    color: $color-primary-darker;
+    background-color: $button__background-color--hover;
```

Comment:
Looking at my local doc site, the hover state for default and transparent buttons appears to be broken.

<img width="561" alt="Screen Shot 2022-05-02 at 9 37 09 AM" src="https://user-images.githubusercontent.com/7595652/166284588-fb12c9b2-8e9f-4b17-b71a-62b2e06f3f1a.png">

If I remove the `ds-c-button--hover` class, the true hover state is still that way.

Reply:
yikes, an extra background attribute snuck in after my testing.. should be resolved, just went through all 3 child systems to be sure.